packages/ti_abusech/_dev/benchmark/system/deploy/docker/files/config.yml

```
@@ -0,0 +1,108 @@
+rules:
+  # ja3_fingerprints data stream
+  - path: /blacklist/ja3_fingerprints.csv
+    methods: ["GET"]
+    responses:
+      - status_code: 200
+        body: |-
+          ################################################################
+          # abuse.ch Suricata JA3 Fingerprint Blacklist (CSV)            #
+          # For Suricata 4.1.0 or newer                                  #
+          # Last updated: 2021-08-03 14:33:44 UTC                        #
+          #                                                              #
+          # Terms Of Use: https://sslbl.abuse.ch/blacklist/              #
+          # For questions please contact sslbl [at] abuse.ch             #
+          ################################################################
+          #
+          # ja3_md5,Firstseen,Lastseen,Listingreason
+          {{- $g := glob "/var/log/corpus-*" -}}
+          {{- range $g -}}
+            {{- file . -}}
+          {{- end -}}
+          b386946a5a44d1ddcc843bc75336dfce,2017-07-14 18:08:15,2019-07-27 20:42:54,Dridex
```

The last line of hard-coded data is required to avoid empty string from being sent to CEL program, which returns an `error.message`